src/test/java/com/google/devtools/build/lib/rules/test/TestTargetPropertiesTest.java

```diff
@@ -52,6 +52,35 @@ public void testTestWithCpusTagHasCorrectLocalResourcesEstimate() throws Excepti
     assertThat(localResourceUsage.getCpuUsage()).isEqualTo(4.0);
   }
 
+  @Test
+  public void testTestResourcesFlag() throws Exception {
+    scratch.file("tests/test.sh", "#!/bin/bash", "exit 0");
+    scratch.file(
+        "tests/BUILD",
+        "sh_test(",
+        "  name = 'test',",
+        "  size = 'medium',",
+        "  srcs = ['test.sh'],",
+        "  tags = ['resources:gpu:4'],",
+        ")");
+    useConfiguration(
+        "--test_resources=memory=10,20,30,40",
```

Comment:
Tests are failing, think you need to update the flag name here to `--default_test_resources`.